src/VisualStudio/CSharp/Test/PersistentStorage/Mocks/AuthorizationServiceMock.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceHub.Framework.Services;
+
+#pragma warning disable CS0067 // events that are never used
+
+namespace Microsoft.CodeAnalysis.UnitTests.WorkspaceServices.Mocks
```

Comment:
📝 Ah, this is a case where the containing folder and namespace should be `Fakes`

```suggestion
namespace Microsoft.CodeAnalysis.UnitTests.WorkspaceServices.Fakes
```